### ISSUE TITLE
HbbTV_DVB "audio sample descriptor type not found, sampling rate not found" error on all contents

### ISSUE THREAD

**Issue by Murmur** [Dash-Industry-Forum/DASH-IF-Conformance#652]
This error `audio sample descriptor type not found, sampling rate not found` error is found pretty much on all content, platforms are happy to playback a dash stream(hbbtv device, pc browser, mobile device). Is this FAIL status a false positive or just `ffmpeg -a:c aac` aac_lc impl does not want to write `AudioSampleBox.sdType, AudioSampleBox.samplingRate` fields?  HbbTV specs explicitly wanted them be present?

* maybe not-well-known ffmpeg or mp4box argument could force values be written to a mp4 file if there is any real reason fields be present.

https://github.com/Dash-Industry-Forum/DASH-IF-Conformance/blob/2ab53f79e410c2b22dd2bf61ebe6e86336cdeef4/HbbTV_DVB/impl/commonHbbTVValidation.php#L185

```
16s limit: https://refapp.hbbtv.org/videos/dashtest/dashjs_1.mpd
original: https://dash.akamaized.net/akamai/bbb_30fps/bbb_30fps.mpd

Section: HbbTV: Section E.2.3, Module: HbbTV_DVB
All info necessary to decode any Segment shall be provided in Initialization Segment
Audio: Sample description type not found
Audio: Sampling rate not found
```

**Comment by dsilhavy** [comment]
Addressed in #659 @Murmur can you please confirm that your problem is fixed on https://staging.conformance.dashif.org/

**Comment by dsilhavy** [comment]
Thanks for testing @Murmur
